<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="2" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="22223" started-at="2020-08-14T18:12:29 IST" finished-at="2020-08-14T18:12:51 IST">
    <groups>
    </groups>
    <test name="Test1" duration-ms="94" started-at="2020-08-14T18:12:29 IST" finished-at="2020-08-14T18:12:29 IST">
      <class name="Priya.HomePage">
        <test-method status="PASS" signature="getDemo(java.lang.String)[pri:0, instance:Priya.HomePage@6392827e]" name="getDemo" duration-ms="14" started-at="2020-08-14T18:12:29 IST" finished-at="2020-08-14T18:12:29 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Check]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getDemo -->
        <test-method status="FAIL" signature="close()[pri:0, instance:Priya.HomePage@6392827e]" name="close" is-config="true" duration-ms="2" started-at="2020-08-14T18:12:29 IST" finished-at="2020-08-14T18:12:29 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at Priya.HomePage.close(HomePage.java:66)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:64)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:364)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:318)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:620)
at org.testng.TestRunner.afterRun(TestRunner.java:850)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- Priya.HomePage -->
    </test> <!-- Test1 -->
    <test name="Test2" duration-ms="21372" started-at="2020-08-14T18:12:30 IST" finished-at="2020-08-14T18:12:51 IST">
      <class name="Priya.validateTitle">
        <test-method status="PASS" signature="titleCheck()[pri:0, instance:Priya.validateTitle@47e2e487]" name="titleCheck" duration-ms="17198" started-at="2020-08-14T18:12:30 IST" finished-at="2020-08-14T18:12:47 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleCheck -->
        <test-method status="PASS" signature="close()[pri:0, instance:Priya.validateTitle@47e2e487]" name="close" is-config="true" duration-ms="4169" started-at="2020-08-14T18:12:47 IST" finished-at="2020-08-14T18:12:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- Priya.validateTitle -->
    </test> <!-- Test2 -->
  </suite> <!-- Suite -->
</testng-results>
